src/store/plugins/sdk.js

```diff
@@ -33,144 +31,28 @@ export default (store) => {
-          }
-          action.deny();
-        };
-
-        const signCbBackground = async (type, aepp, action) => {
-          const { method, params } = action;
-          if (
-            (await store.dispatch('permissions/checkPermissions', {
-              host: getAeppUrl(aepp).hostname,
-              method,
-              params: params?.txObject?.params,
-            }))
-            || (await showPopup(aepp, type, params).then(
-              () => true,
-              () => false,
-            ))
-          ) {
-            action.accept.apply(null, [
-              ...(method === 'message.sign' ? [] : [null]),
-              { onAccount: { sign: () => {}, address: () => {} } },
-            ]);
-            return;
-          }
-          action.deny();
-        };
-
-        sdk = await RpcWallet.compose({
+        sdk = await Universal.compose({
           methods: {
             getApp(aeppUrl) {
               return new App(aeppUrl);
             },
-            async address(...args) {
+            async address() {
               const { address } = activeAccount.value;
-              const app = args.pop();
-              if (
-                app instanceof App
-                && !(await store.dispatch('permissions/requestAddressForHost', {
-                  host: app.host.host,
-                  name: app.host.hostname,
-                  address,
-                  connectionPopupCb: async () => (IS_EXTENSION_BACKGROUND
-                    ? showPopup(app.host.href, 'connectConfirm')
-                    : openModal(MODAL_CONFIRM_CONNECT, {
-                      app: {
-                        name: app.host.hostname,
-                        icons: [],
-                        protocol: app.host.protocol,
-                        host: app.host.host,
-                        url: app.host.href,
-                      },
-                    })),
-                }))
-              ) return Promise.reject(new Error('Rejected by user'));
               return address;
             },
             sign: (data) => (IS_EXTENSION_BACKGROUND
               ? Crypto.sign(data, activeAccount.value.secretKey)
               : store.dispatch('accounts/sign', data)),
             ...(IS_EXTENSION_BACKGROUND ? {} : {
-              signTransaction: (txBase64, opt) => (opt.onAccount
+              signTransaction: (txBase64, opt) => (typeof opt.onAccount === 'object')
                 ? opt.onAccount.sign()
-                : store.dispatch('accounts/signTransaction', { txBase64, opt })),
+                : store.dispatch('accounts/signTransaction', { txBase64, opt }),
             }),
           },
         })({
           nodes: [{ name: activeNetwork.name, instance: await Node({ url: activeNetwork.url }) }],
           compilerUrl: activeNetwork.compilerUrl,
           name: 'Superhero',
-          onConnection: cbAccept,
-          onDisconnect(_, client) {
-            client.disconnect();
```

Comment:
This is not longer being propagated.

Reply:
With the new SDK version, there is no need to propagate disconnect. @davidyuk

Reply:
Well, the `superhero-ui` now doesn't reload the state if the wallet is resetting.

Reply:
Good catch. Let me check.

Reply:
@CedrikNikita Fixed it.

Reply:
> there is no need to propagate disconnect

This part should be working the same as in sdk@12